frontend/src/styles/components/commentform.css

```diff
@@ -17,6 +17,7 @@
 }
 
 .length {
```

Comment:
i can't tell but is there extra space after the dot? is the styling applied? if not please remove the space

Reply:
I don't think there's a space